### ISSUE TITLE
Strange errors since updating to 1.04

### ISSUE THREAD

**Issue by oschwald** [houseabsolute/DateTime-Locale#3]
After upgrading from 0.45 to 1.04, we are receiving errors like this in our logs. They appear to happen at random.

```
Bareword found where operator expected at (eval 15681) line 62, near "516c"^J') called at (eval 15681) line 62
eval '1} {0}",
  datetime_format_short => "{1} {0}",
  day_format_abbreviated => [
    "\\N{U+5468}\\N{U+4e00}",
    "\\N{U+5468}\\N{U+4e8c}",
    "\\N{U+5468}\\N{U+4e09}",
    "\\N{U+5468}\\N{U+56db}",
    "\\N{U+5468}\\N{U+4e94}",
    "\\N{U+5468}\\N{U+516d}",
    "\\N{U+5468}\\N{U+65e5}"
  ],
  day_format_narrow => [
    "\\N{U+4e00}",
    "\\N{U+4e8c}",
    "\\N{U+4e09}",
    "\\N{U+56db}",
    "\\N{U+4e94}",
    "\\N{U+516d}",
    "\\N{U+65e5}"
  ],
  day_format_wide => [
    "\\N{U+661f}\\N{U+671f}\\N{U+4e00}",
    "\\N{U+661f}\\N{U+671f}\\N{U+4e8c}",
    "\\N{U+661f}\\N{U+671f}\\N{U+4e09}",
    "\\N{U+661f}\\N{U+671f}\\N{U+56db}",
    "\\N{U+661f}\\N{U+671f}\\N{U+4e94}",
    "\\N{U+661f}\\N{U+671f}\\N{U+516d}",
    "\\N{U+661f}\\N{U+671f}\\N{U+65e5}"
  ],
  day_stand_alone_abbreviated => [
    "\\N{U+5468}\\N{U+4e00}",
    "\\N{U+5468}\\N{U+4e8c}",
    "\\N{U+5468}\\N{U+4e09}",
    "\\N{U+5468}\\N{U+56db}",
    "\\N{U+5468}\\N{U+4e94}",
    "\\N{U+5468}\\N{U+516d}",
    "\\N{U+5468}\\N{U+65e5}"
  ],
  day_stand_alone_narrow => [
    "\\N{U+4e00}",
    "\\N{U+4e8c}",
    "\\N{U+4e09}",
    "\\N{U+56db}",
    "\\N{U+4e94}",
    "\\N{U+516d}",
    "\\N{U+65e5}"
  ],
  day_stand_alone_wide => [
    "\\N{U+661f}\\N{U+671f}\\N{U+4e00}",
    "\\N{U+661f}\\N{U+671f}\\N{U+4e8c}",
    "\\N{U+661f}\\N{U+671f}\\N{U+4e09}",
    "\\N{U+661f}\\N{U+671f}\\N{U+56db}",
    "\\N{U+661f}\\N{U+671f}\\N{U+4e94}",
    "\\N{U+661f}\\N{U+671f}\\N{U+516d}",
    "\\N{U+661f}\\N{U+671f}\\N{U+65e5}"
  ],
  era_abbreviated => [
    "\\N{U+516c}\\N{U+5143}\\N{U+524d}",
    "\\N{U+516c}\\N{U+5143}"
  ],
  era_narrow => [
    "\\N{U+516c}\\N{U+5143}\\N{U+524d}",
    "\\N{U+516c}\\N{U+5143}"
  ],
  era_wide => [
    "\\N{U+516c}\\N{U+5143}\\N{U+524d}",
    "\\N{U+516c}\\N{U+5143}"
  ],
  first_day_of_week => 1,
  glibc_date_1_format => "%a %b %e %H:%M:%S %Z %Y",
  glibc_date_format => "/%d/%y",
  glibc_datetime_format => "%a %b %e %H:%M:%S %Y",
  glibc_time_12_format => "%I:%M:%S %p",
  glibc_time_format => "%H:%M:%S",
  language => "Chinese",
  month_format_abbreviated => [
    "1\\N{U+6708}",
    "2\\N{U+6708}",
    "3\\N{U+6708}",
    "4\\N{U+6708}",
    "5\\N{U+6708}",
    "6\\N{U+6708}",
    "7\\N{U+6708}",
    "8\\N{U+6708}",
    "9\\N{U+6708}",
    "10\\N{U+6708}",
    "11\\N{U+6708}",
    "12\\N{U+6708}"
  ],
  month_format_narrow => [
    1,
    2,
    3,
    4,
    5,
    6,
    7,
    8,
    9,
    10,
    11,
    12
  ],
  month_format_wide => [
    "\\N{U+4e00}\\N{U+6708}",
    "\\N{U+4e8c}\\N{U+6708}",
    "\\N{U+4e09}\\N{U+6708}",
    "\\N{U+56db}\\N{U+6708}",
    "\\N{U+4e94}\\N{U+6708}",
    "\\N{U+516d}\\N{U+6708}",
    "\\N{U+4e03}\\N{U+6708}",
    "\\N{U+516b}\\N{U+6708}",
    "\\N{U+4e5d}\\N{U+6708}",
    "\\N{U+5341}\\N{U+6708}",
    "\\N{U+5341}\\N{U+4e00}\\N{U+6708}",
    "\\N{U+5341}\\N{U+4e8c}\\N{U+6708}"
  ],
  month_stand_alone_abbreviated => [
    "1\\N{U+6708}",
    "2\\N{U+6708}",
    "3\\N{U+6708}",
    "4\\N{U+6708}",
    "5\\N{U+6708}",
    "6\\N{U+6708}",
    "7\\N{U+6708}",
    "8\\N{U+6708}",
    "9\\N{U+6708}",
    "10\\N{U+6708}",
    "11\\N{U+6708}",
    "12\\N{U+6708}"
  ],
  month_stand_alone_narrow => [
    1,
    2,
    3,
    4,
    5,
    6,
    7,
    8,
    9,
    10,
    11,
    12
  ],
  month_stand_alone_wide => [
    "\\N{U+4e00}\\N{U+6708}",
    "\\N{U+4e8c}\\N{U+6708}",
    "\\N{U+4e09}\\N{U+6708}",
    "\\N{U+56db}\\N{U+6708}",
    "\\N{U+4e94}\\N{U+6708}",
    "\\N{U+516d}\\N{U+6708}",
    "\\N{U+4e03}\\N{U+6708}",
    "\\N{U+516b}\\N{U+6708}",
    "\\N{U+4e5d}\\N{U+6708}",
    "\\N{U+5341}\\N{U+6708}",
    "\\N{U+5341}\\N{U+4e00}\\N{U+6708}",
    "\\N{U+5341}\\N{U+4e8c}\\N{U+6708}"
  ],
  name => "Chinese",
  native_language => "\\N{U+4e2d}\\N{U+6587}",
  native_name => "\\N{U+4e2d}\\N{U+6587}",
  native_script => undef,
  native_territory => undef,
  native_variant => undef,
  quarter_format_abbreviated => [
    "1\\N{U+5b63}\\N{U+5ea6}",
    "2\\N{U+5b63}\\N{U+5ea6}",
    "3\\N{U+5b63}\\N{U+5ea6}",
    "4\\N{U+5b63}\\N{U+5ea6}"
  ],
  quarter_format_narrow => [
    1,
    2,
    3,
    4
  ],
  quarter_format_wide => [
    "\\N{U+7b2c}\\N{U+4e00}\\N{U+5b63}\\N{U+5ea6}",
    "\\N{U+7b2c}\\N{U+4e8c}\\N{U+5b63}\\N{U+5ea6}",
    "\\N{U+7b2c}\\N{U+4e09}\\N{U+5b63}\\N{U+5ea6}",
    "\\N{U+7b2c}\\N{U+56db}\\N{U+5b63}\\N{U+5ea6}"
  ],
  quarter_stand_alone_abbreviated => [
    "1\\N{U+5b63}\\N{U+5ea6}",
    "2\\N{U+5b63}\\N{U+5ea6}",
    "3\\N{U+5b63}\\N{U+5ea6}",
    "4\\N{U+5b63}\\N{U+5ea6}"
  ],
  quarter_stand_alone_narrow => [
    1,
    2,
    3,
    4
  ],
  quarter_stand_alone_wide => [
    "\\N{U+7b2c}\\N{U+4e00}\\N{U+5b63}\\N{U+5ea6}",
    "\\N{U+7b2c}\\N{U+4e8c}\\N{U+5b63}\\N{U+5ea6}",
    "\\N{U+7b2c}\\N{U+4e09}\\N{U+5b63}\\N{U+5ea6}",
    "\\N{U+7b2c}\\N{U+56db}\\N{U+5b63}\\N{U+5ea6}"
  ],
  script => undef,
  territory => undef,
  time_format_full => "zzzz ah:mm:ss",
  time_format_long => "z ah:mm:ss",
  time_format_medium => "ah:mm:ss",
  time_format_short => "ah:mm",
  variant => undef,
  version => 29
}
__[ zh-Hans-HK ]__
{
  am_pm_abbreviated => [
    "\\N{U+4e0a}\\N{U+5348}",
    "\\N{U+4e0b}\\N{U+5348}"
  ],
  available_formats => {
    E => "ccc",
    EHm => "EHH:mm",
    EHms => "EHH:mm:ss",
    Ed => "d\\N{U+65e5}E",
    Ehm => "E ah:mm",
    Ehms => "E ah:mm:ss",
    Gy => "Gy\\N{U+5e74}",
    GyMMM => "Gy\\N{U+5e74}M\\N{U+6708}",
    GyMMMEd => "Gy\\N{U+5e74}M\\N{U+6708}d\\N{U+65e5}E",
    GyMMMd => "Gy\\N{U+5e74}M\\N{U+6708}d\\N{U+65e5}",
    H => "H\\N{U+65f6}",
    HHmm => "HH:mm",
    Hm => "HH:mm",
    Hms => "HH:mm:ss",
    Hmsv => "v HH:mm:ss",
    Hmv => "v HH:mm",
    M => "M\\N{U+6708}",
    MEd => "E, d/M",
    MMM => "LLL",
    MMMEd => "M\\N{U+6708}d\\N{U+65e5}E",
    MMMMd => "M\\N{U+6708}d\\N{U+65e5}",
    MMMMdd => "M\\N{U+6708}d\\N{U+65e5}",
    MMMd => "M\\N{U+6708}d\\N{U+65e5}",
    MMdd => "dd/MM",
    Md => "d/M",
    d => "d\\N{U+65e5}",
    h => "ah\\N{U+65f6}",
    hm => "ah:mm",
    hms => "ah:mm:ss",
    hmsv => "v ah:mm:ss",
    hmv => "v ah:mm",
    ms => "mm:ss",
    y => "y\\N{U+5e74}",
    yM => "M/y",
    yMEd => "d/M/y\\N{U+ff08}E\\N{U+ff09}",
    yMM => "MM/y",
    yMMM => "y\\N{U+5e74}M\\N{U+6708}",
    yMMMEd => "y\\N{U+5e74}M\\N{U+6708}d\\N{U+65e5}E",
    yMMMM => "y\\N{U+5e74}M\\N{U+6708}",
    yMMMd => "y\\N{U+5e74}M\\N{U+6708}d\\N{U+65e5}",
    yMd => "d/M/y",
    yQQQ => "y\\N{U+5e74}\\N{U+7b2c}Q\\N{U+5b63}\\N{U+5ea6}",
    yQQQQ => "y\\N{U+5e74}\\N{U+7b2c}Q\\N{U+5b63}\\N{U+5ea6}"
  },
  code => "zh-Hans-HK",
  date_format_full => "y\\N{U+5e74}M\\N{U+6708}d\\N{U+65e5}EEEE",
  date_format_long => "y\\N{U+5e74}M\\N{U+6708}d\\N{U+65e5}",
  date_format_medium => "y\\N{U+5e74}M\\N{U+6708}d\\N{U+65e5}",
  date_format_short => "d/M/yy",
  datetime_format_full => "{1}' at /opt/perl5.20.2/lib/site_perl/5.20.2/DateTime/Locale/Data.pm line 8492
DateTime::Locale::Data::locale_data('zh') called at /opt/perl5.20.2/lib/site_perl/5.20.2/DateTime/Locale.pm line 278
DateTime::Locale::_locale_object_for('DateTime::Locale', 'zh') called at /opt/perl5.20.2/lib/site_perl/5.20.2/DateTime/Locale.pm line 213
DateTime::Locale::load(undef, 'zh') called at /opt/perl5.20.2/lib/site_perl/5.20.2/MM/Website/Util.pm line 279
```

**Comment by autarch** [comment]
It seems like something is somehow messing with the `DATA` handle in `DateTime::Locale::Data`. I don't know if there's any way I can reasonably diagnose that, so I'll look into another mechanism for dealing with this data.

**Comment by autarch** [comment]
Hopefully fixed in 1.05.
